### PR TITLE
docs: document major/minor tagging

### LINKED ISSUE #91: feat: add support for tagging vX, vX.X tags
When release-please creates a release, it would be nice if it also created a `v2` and `v2.2` tag (this is particularly useful for GitHub Actions, which do not support install ranges).

Refs: https://github.com/nodejs/gyp-next/pull/75

CC: @targos,  @bharathkkb

### PR DESCRIPTION
**Pull request by bcoe**
Documents chaining together `actions/checkout` with `release-please-action` to create major/minor tags.

Fixes #91

### CODE REVIEW

**Comment by bcoe** [comment]
CC: @targos (we starting doing this on `relese-please` itself the last few releases, let me know if it causes any issues).

File: README.md [hunk 1]
```diff
@@ -166,6 +166,53 @@ jobs:
...
+## Creating major/minor tags
+
+If you are using release-please to publish a GitHub acton, you will
+likely want to tag a major and minor tag during a release, i.e., if you
+are releasing `v2.8.3`, you will also tag `v2` nd `v2.8`. This allows your
+users to pin to `v2`, and get updates to your library without updating their
+workflows.
+
+The `release-please-action` has the outputs `major`, `minor`, and
+`release_created` to facilitate this. These outputs can bey used conditionally,
```
Comment (bharathkkb) [inline README.md:179]:
```suggestion
`release_created` to facilitate this. These outputs can be used conditionally,
```

File: README.md [hunk 2]
```diff
@@ -166,6 +166,53 @@ jobs:
 > So that you can keep 2FA enabled for npm publications, we recommend setting
 `registry-url` to your own [Wombat Dressing Room](https://github.com/GoogleCloudPlatform/wombat-dressing-room) deployment.
 
+
+## Creating major/minor tags
+
+If you are using release-please to publish a GitHub acton, you will
+likely want to tag a major and minor tag during a release, i.e., if you
+are releasing `v2.8.3`, you will also tag `v2` nd `v2.8`. This allows your
```
Comment (bharathkkb) [inline README.md:174]:
```suggestion
are releasing `v2.8.3`, you will also want to update tags `v2` and `v2.8`. This allows your
```

**Review by bharathkkb — APPROVED** [review]
Thanks for putting this together!

File: README.md [hunk 3]
```diff
@@ -166,6 +166,53 @@ jobs:
...
+          git remote add gh-token "https://${{ secrets.GITHUB_TOKEN}}@github.com/google-github-actions/release-please-action.git"
+          git tag -d v${{ steps.release.outputs.major }} || true
+          git tag -d v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
+          git push origin :v${{ steps.release.outputs.major }} || true
+          git push origin :v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
+          git tag -a v${{ steps.release.outputs.major }} -m "Release v${{ steps.release.outputs.major }}"
+          git tag -a v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} -m "Release v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}"
+          git push origin v${{ steps.release.outputs.major }}
+          git push origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}
+        if: ${{ steps.release.outputs.release_created }}
```
Comment (bharathkkb) [inline README.md:213]: For readability it maybe better to move the conditional up

File: README.md [hunk 4]
```diff
@@ -166,6 +166,53 @@ jobs:
...
```
Comment (bcoe) [inline README.md:213]:
```suggestion
      - name: tag major and patch versions
         if: ${{ steps.release.outputs.release_created }}
         run: |
           git config user.name github-actions[bot]
           git config user.email 41898282+github-actions[bot]@users.noreply.github.com
           git remote add gh-token "https://${{ secrets.GITHUB_TOKEN}}@github.com/google-github-actions/release-please-action.git"
           git tag -d v${{ steps.release.outputs.major }} || true
           git tag -d v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
           git push origin :v${{ steps.release.outputs.major }} || true
           git push origin :v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
           git tag -a v${{ steps.release.outputs.major }} -m "Release v${{ steps.release.outputs.major }}"
           git tag -a v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} -m "Release v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}"
           git push origin v${{ steps.release.outputs.major }}
           git push origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}
```

**Pull request merged**